go/vt/vtgate/vstream_manager_test.go

```diff
@@ -333,6 +333,57 @@ func TestVStreamMulti(t *testing.T) {
 	}
 }
 
+func TestVStreamMetrics(t *testing.T) {
```

Comment:
I think that we should either change the test name or the test as currently it only seems to be testing the two new metrics rather than all metrics.